llmeval/launch_eval.py

```diff
@@ -66,6 +66,12 @@ def main():
         help="location where evaluation json files will be written",
         required=True,
     )
+    parser.add_argument(
+        "--symlink_path",
+        type=str,
+        help="location where to store symlinks for models to evaluate",
+        required=True,
```

Comment:
Can we make it optional? Currently evals can also be launched with public models on HF such as `Qwen/Qwen2.5-1.5B`, this would not be required in such cases.